### DISCUSSION TITLE
exit code 1

### DISCUSSION THREAD

**Discussion by balrod16** [aclap-dev/video-downloadhelper#364] [Help and Q&A]
### Your question

error shows

Exit code: 1
Error: Exit code: 1

    at as.receive (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:13157)
    at chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:1:42624

just updated both the companion app and windows and this error started appearing when trying to download some videos, as a side note those who show the file size work fine, but if it shows the resolution or how long the video is, it automatically pop ups the error (HLS download?)
 already tried disabling windows firewall and adding the companion app to defender exceptions list

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: es-ES
license: accepted
key (only 16 first characters): 2729e1d35fc34a3b
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\balro",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
Do you have a link to share where this happens, is it a public video. If it private share the hit details information as text.

**Comment by Mattmjs** [comment]
Also try the beta extension and see if problem is the same or not.

**Comment by balrod16** [comment]
sorry for the late answer, i have no idea how to extract the video information, so i looked in the forum and saw someone opened a thread with the same error in the same website, tried the beta extension (beta 8.2.2.9) and didnt work either, seeing it worked for you, ill just wait for an update

**Comment by balrod16** [comment]
tried again downloading the file, seems to work fine, after a couple of minutes the download stops without an error and the file its imcomplete, tried downloading it again and the same error pops, website its missav.com btw

**Reply by Mattmjs** [reply]
See my response on similar discussion: https://github.com/aclap-dev/video-downloadhelper/discussions/375#discussioncomment-9018041
